AzureFunction/Utilities.cs

```diff
@@ -0,0 +1,160 @@
+﻿namespace Microsoft.Azure.Pipelines.EvaluateArtifactPolicies
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Diagnostics;
+    using System.Globalization;
+    using System.IO;
+    using System.Linq;
+    using System.Text.RegularExpressions;
+
+    using Microsoft.Azure.Pipelines.EvaluateArtifactPolicies.Models;
+    using Microsoft.Azure.Pipelines.EvaluateArtifactPolicies.Request;
+    using Microsoft.Extensions.Logging;
+    using Newtonsoft.Json;
+    using Newtonsoft.Json.Linq;
+
+    using WebJobsExecutionContext = WebJobs.ExecutionContext;
+
+    public static class Utilities
+    {
+        private const string ImageProvenanceFileName = "ImageProvenance.json";
+        private const string PolicyFileName = "Policies.rego";
+        private const string OutputResultFileName = "Output.txt";
+
+        public static IEnumerable<string> ExecutePolicyCheck(
+            WebJobsExecutionContext executionContext,
+            ILogger log,
+            string imageProvenance,
+            string policy,
+            TaskLogger taskLogger,
+            out string outputLog)
+        {
+            string folderName = string.Format("Policy-{0}", executionContext.InvocationId.ToString("N"));
```

Comment:
Does each request has different invocation Id? or is it common per azure function instance (instance handles multiple requests?

Reply:
Different invocation id per call